client/state/data-layer/wpcom-http/pipeline/remove-duplicate-gets/index.js

```diff
@@ -41,6 +41,18 @@ export const clearQueue = () => {
  */
 const isGetRequest = ( request ) => 'GET' === get( request, 'method', '' ).toUpperCase();
 
+/**
+ * Returns all elements that exist in any of the two arrays at least once,
+ *
+ * @param {Array} a First array
+ * @param {Array} b Second array
+ * @returns {Array} Array of elements that exist in at least one of the arrays
+ */
+const unionWith = ( a = [], b = [] ) =>
+	Array.from(
+		new Set( [ ...a, ...b.filter( ( x ) => a.findIndex( ( y ) => isEqual( x, y ) ) === -1 ) ] )
```

Comment:
Why is the set necessary if we're already filtering `b`?

Reply:
Good question, I think we can also ditch the `Array.from` I guess. Removed in https://github.com/Automattic/wp-calypso/pull/51977/commits/f06a7e187049e7c1e46732b1a1610bc3d61046e3 and https://github.com/Automattic/wp-calypso/pull/51977/commits/530229d99d6e6bfb7e874588a26359c132d0a79b